File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(string s) {
    string answer = "";
    
    bool bFirst = true;
    for(char c : s)
    {
        if(c == ' ') 
        {
            answer += tolower(c);
            bFirst = true;
            continue;
        }
        
        if(bFirst)
        {
            bFirst = false;
            if(c - '0' >= 0 && c - '0' <= 9) answer += c;
            else answer += toupper(c);
        }
        else
        {
            answer += tolower(c);
        }
    }
    
    return answer;
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cs
using System;

public class Solution {
    public string solution(string s) {
        string answer = "";
        
        bool isFirst = true;
        for(int i = 0; i < s.Length; i++)
        {
            if(s[i] == ' ')
            {
                isFirst = true;
                answer += s[i];
                continue;
            }
            
            if(isFirst)
            {
                isFirst = false;
                var c = s[i];
                if(!('0' <= c && c <= '9'))
                {
                    c = char.ToUpper(c);
                }
                
                answer += c;
                continue;
            }
                      
            answer += char.ToLower(s[i]);
        }
        
        return answer;
    }
}